NGitLab/IIssueClient.cs

```diff
@@ -180,4 +180,20 @@ public interface IIssueClient
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
     Task<Issue> CloneAsync(int projectId, int issueIid, IssueClone issueClone, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Get Participants
+    /// </summary>
+    /// <param name="projectId">The project id</param>
+    /// <param name="issueIid">The id of the issue in the project's scope.</param>
+    /// <returns>The participants of the issue.</returns>
+    IEnumerable<Participant> GetParticipants(int projectId, int issueIid);
```

Comment:
As they are new methods, you should use `ProjectId` as type instead of regular int.

Then use `projectId.ValueAsUriParameter()` to construct the URLs in the client implementations